.github/workflows/cla_check.yml

```diff
@@ -0,0 +1,34 @@
+name: CLA check
+
+# check if the PR author has signed the CLA
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+      - cla_check
+
+jobs:
+  cla_check_job:
+    runs-on:  ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-18.04]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Install Dependecies
+        run: |
+          npm install googleapis@39
```

Comment:
Why cannot this be installed through package.json?

Reply:
This file is to be removed

Reply:
What about this?